spread.yaml

```diff
@@ -149,6 +149,14 @@ backends:
             - ubuntu-18.04-64:
                 image: ubuntu-1804-64-virt-enabled
                 workers: 1
+            - ubuntu-19.10-64:
+                # FIXME: use 19.10 image
+                image: ubuntu-1804-64-virt-enabled
```

Comment:
CC @sergiocazzolato Could you please help set those up?

Reply:
No, we actually need host system to match nested vm image, it needs to be same ubuntu release. The point of preseeding is that we run snand and some portions of seeding on the host with snap-preseed command; this generates apparmor/secomp profiles for the target image. It will only reliably work if the kernel of the host and target vm image are identical in terms of apparmor features (there are differences between ubuntu releases).

Without this the system-key of the booted VM will be different than the one created during pre-seeding, meaning snapd on firstboot will re-create all security profiles. I've a specific check for system-key in the spread test to detect such case, currently it is commented out because it fails on 19.04 and 20.04 due to this ubuntu mismatch.

Reply:
@stolowski ah, ok, makes sense, letme create those image in that case. For ubuntu 20.04 is it possible to use the one already created called ubuntu-2004-64-virt-uefi-enabled
Thanks for the explanation

Reply:
@stolowski I also created the image: ubuntu-1910-64-virt-enabled